arrocke/gloss-translation
# chore: feature flag for comments tab

## What has changed

* System for managing feature flags on the web project.
* Conditionally render the comments tab based on the feature flag state and user permissions
* Docs for how feature flags work




## Connected Issues

closes #304 



## QA Steps

1. Log in as platform admin
1. Try adding `NX_FEAT_COMMENTS=true` to `.env.local` in the web package - the comments tab should show
2. Try adding `?_f=comments` to the end of the url - the comments tab should show
3. Otherwise there should be no comments.
4. Try the same without being logged in. Confirm that comments tab never shows



## Post-Deployment
